server/app/boot_jobs.rb

```diff
@@ -5,6 +5,5 @@
 unless ENV['RACK_ENV'] == 'test' || ENV['NO_MONGO_PUBSUB']
```

Comment:
It's a badly named hack, but it seems to only be used by the rake and `bin/kontena-console` utilities to avoid running background tasks/pubsub stuff.

```
./app/boot_jobs.rb:unless ENV['RACK_ENV'] == 'test' || ENV['NO_MONGO_PUBSUB']
./lib/tasks/reset_admin.rake:    ENV['NO_MONGO_PUBSUB'] = 'true'
./bin/kontena-console:ENV['NO_MONGO_PUBSUB'] = 'true'
```